### ISSUE TITLE
Functional test/functional/tm/ltm/test_virtual.py test errors out while trying to delete sys/db entry

### ISSUE THREAD

**Issue by pjbreaux** [F5Networks/f5-common-python#594]
Before the test is run, a snapshot is taken of the device to get the resource state. After the test is over, another snapshot is taken and compared against the pre-test snapshot. The diff of resources are deleted, but one resource exists that cannot be deleted.

```
==================================== ERRORS ====================================
___________ ERROR at teardown of test_profiles_CE_check_load_params ____________

    def teardown():
        after_snapshot = register_device(mgmt_root)
        diff = set(after_snapshot) - set(before_snapshot)
        for item in diff:
>           after_snapshot[item].delete()

conftest.py:173: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <f5.bigip.tm.sys.db.Db object at 0x7f6c7b2fb990>, kwargs = {}

    def delete(self, **kwargs):
        '''Delete is not supported for db resources.

            :raises: UnsupportedOperation
            '''
        raise UnsupportedOperation(
>           "DB resources doesn't support delete, only load and refresh"
        )
E       UnsupportedOperation: DB resources doesn't support delete, only load and refresh

f5/bigip/tm/sys/db.py:69: UnsupportedOperation
============== 445 passed, 39 skipped, 1 error in 106.67 seconds ===============
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```

**Comment by zancas** [comment]
@pjbreaux This is an 11.5.4 issue.   I wonder if we should organize version-skipped tests with a new github label?  Ideas?

**Comment by pjbreaux** [comment]
Done!

**Comment by pjbreaux** [comment]
Fixed by PR #588
